concourse/docker/Dockerfile

```diff
@@ -9,5 +9,9 @@ RUN apt-get update \
   # cleanup
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
+
+COPY dev-requirements.txt /tmp/dev-requirements.txt
```

Comment:
Why not the `requirements.txt`? For unit tests, the dependencies would be needed, wouldn't they?

Reply:
I just checked. The `requirements.txt` are installed at runtime of the pipeline. There are pros and cons for both:
- Installing it in the Dockerfile makes the pipeline faster, but the requirements will not reflect the current requirements (rather the requirements from the time the image was built).
- Doing it at runtime slows down the pipeline, but they will be always up to date.